sdk/batch/Azure.ResourceManager.Batch/api/Azure.ResourceManager.Batch.netstandard2.0.cs

```diff
@@ -1202,4 +1208,43 @@ public enum NodeCommunicationMode
         Classic = 1,
         Simplified = 2,
     }
+    public partial class OSDisk
+    {
+        public OSDisk() { }
+        public Azure.ResourceManager.Batch.Models.BatchDiskCachingType? Caching { get { throw null; } set { } }
+        public int? DiskSizeGB { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.BatchDiffDiskPlacement? EphemeralOSDiskPlacement { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.BatchStorageAccountType? ManagedDiskStorageAccountType { get { throw null; } set { } }
+        public bool? WriteAcceleratorEnabled { get { throw null; } set { } }
+    }
+    public partial class SecurityProfile
+    {
+        public SecurityProfile() { }
+        public bool? EncryptionAtHost { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.SecurityType? SecurityType { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.UefiSettings UefiSettings { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SecurityType : System.IEquatable<Azure.ResourceManager.Batch.Models.SecurityType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public SecurityType(string value) { throw null; }
+        public static Azure.ResourceManager.Batch.Models.SecurityType TrustedLaunch { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Batch.Models.SecurityType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Batch.Models.SecurityType left, Azure.ResourceManager.Batch.Models.SecurityType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Batch.Models.SecurityType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Batch.Models.SecurityType left, Azure.ResourceManager.Batch.Models.SecurityType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class UefiSettings
+    {
+        public UefiSettings() { }
+        public bool? SecureBootEnabled { get { throw null; } set { } }
```

Comment:
Could we rename this property to `IsSecureBootEnabled`?

Reply:
updated